python/GangaPanda/Lib/Panda/Panda.py

```diff
@@ -52,6 +53,19 @@
     logger.error("Failed to load AGIS info from http://atlas-agis-api.cern.ch/request/site/query/list/?json&state=ACTI\
 VE&rc_site_state=ACTIVE")
 
+##### fix by jschovan, use outputs of AGIS collector
+def load_json(filename):
+    result=None
+    f=open(filename, 'r')
+    result=json.loads(f.read())
+    f.close()
+    return result
+
+pandaresources_json = '/tmp/agis_pandaresources.json'
```

Comment:
Also, is it worth trying to grab this info from within Ganga or stick with it being created externally?